environments/payu-dev/config.sh

```diff
@@ -0,0 +1,18 @@
+### config.sh MUST provide the following:
+### $FULLENV
+###
+### Arrays used by the build system (optional, can be empty)
+### rpms_to_remove
+### replace_from_apps
+### outside_commands_to_include
+### outside_files_to_copy
+
+### Version settings
+export ENVIRONMENT=payu-dev
+export FULLENV="${ENVIRONMENT}"
+
+declare -a rpms_to_remove=()
+declare -a replace_from_apps=()
+declare -a outside_commands_to_include=( "pbs_tmrsh" )
+declare -a outside_files_to_copy=()
+declare -a replace_with_external=()
```

Comment:
Yes, so these arrays are used in the `build.sh` scripts. For example, the `outside_commands_to_include` array is used here which sets up symlinks to run a launcher script: https://github.com/ACCESS-NRI/model-release-condaenv/blob/32399ec7f63996dd199bbc3b24f79c202c1a6186/scripts/build.sh#L132-L135
I think they persist only because `config.sh` is run using `source` so it runs in the same shell as the build scripts.

Reply:
Gotcha :) thanks!